networkit/stats.pyx

```diff
@@ -1,3 +1,5 @@
+# distutils: language=c++
```

Comment:
Why do we use this directive in favor of a command line switch?

Reply:
Using a command line switch is marked as deprecated (so also output from above). 

```
Warning: passing language='c++' to cythonize() is deprecated. Instead, put "# distutils: language=c++" in your .pyx or .pxd files
```
Personally I would favor the command line switch though ...

Reply:
Okay, if this causes a warning, we should indeed use the annotation.